### DISCUSSION TITLE
Request to add official async NATS client for Rust

### DISCUSSION THREAD

**Discussion by 1tang2bang92** [nats-io/nats.rs#1297] [General]
Hello,

I've been looking at the Rust download page on the official NATS website (https://nats.io/download/rust/). It appears that this page is currently missing a link to the async NATS client.

To make it easier for Rust developers to use NATS, I would like to request that a link to the official async NATS client be added to this page. This would help increase adoption of NATS in the Rust ecosystem.

The client link that could be added is: https://github.com/nats-io/nats.rs

Thank you for considering this request. I appreciate your efforts in improving the NATS community resources.

Best regards,
1tang2bang92

**Comment by Jarema** [comment]
Hey!
Thanks for letting us know.

We will update the page, unless you are interested in contribution :).

**Reply by 1tang2bang92** [reply]
Please provide a github link to your webpage and I'll try to contribute when I have time.
